src/main/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImpl.java

```diff
@@ -18,6 +19,7 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 
+import static java.lang.String.valueOf;
```

Comment:
are these additions accidental?

Reply:
Ah it was in at one point but forgot to CTRL-O.  Will do so on my next PR, thanks.